modules/quickstart-sap/pages/qs-sap-intro.adoc

```diff
@@ -3,7 +3,8 @@
 
 **Publication Date:** {docdate}
 
-This guide shows you the fastest way to get {productname} up and running to manage an SAP cluster.
-It guides you through setting up a single {productname} Server, installing the SAP cluster, registering clients, and setting up a content lifecycle for managing them.
+This guide shows you how to use {productname} to install and configure an SAP cluster.
```

Comment:
I should be `configure a SAP cluster`, right?
And I would add actually add `...configure a SAP cluster with High Availability feature` or similar